app/mailers/report_mailer.rb

```diff
@@ -14,6 +14,32 @@ def budget_status
     )
   end
 
+  def annual_report(year, to_email)
+    proposals = Proposal.approved.where(client_data_type: "Ncr::WorkOrder")
+        proposals = proposals.select {|p| p.client_data.fiscal_year == year}
+        csv_string = CSV.generate do |csv|
+          csv << ["Amount", "Date Approved", "Org Code", "CL#", 
+            "Budget Activity", "SOC", "Function Code", "Building #", 
+            "Vendor", "Description", "Requestor", "Approver"]
+          for p in proposals
```

Comment:
this report is specific to NCR. Should that be reflected in the method name somehow? I realize there's poor precedent for the reports since they are all NCR at this point.